app/routes/signups.js

```diff
@@ -5,6 +5,7 @@ const router = express.Router(); // eslint-disable-line new-cap
 const helpers = require('../../lib/helpers');
 const NotFoundError = require('../exceptions/NotFoundError');
 const UnprocessibleEntityError = require('../exceptions/UnprocessibleEntityError');
+const Promise = require('bluebird'); // eslint-disable-line no-unused-vars
```

Comment:
Could you please add a comment why we need to import bluebird here? I just worry that it might get deleted someday, on incorrect assumption that it's not used.